Gemfile

```diff
@@ -1,8 +1,10 @@
 source 'https://rubygems.org'
 
 group :test do
-  gem 'cucumber', "=1.3.20"
-  gem 'aruba', "=0.6.1"
-  gem 'rspec', "=2.14.1"
+  gem 'gem-shut-the-fuck-up'
```

Comment:
Please don't! :-) I'd rather not having it as a dependency! You can move it into the build script if needed. Users should not be forced to install it if they don't want to.